main.go

```diff
@@ -1,69 +1,59 @@
-// A minimal example of how to include Prometheus instrumentation.
 package main
 
 import (
 	"crypto/tls"
 	"encoding/xml"
 	"flag"
+	"github.com/joho/godotenv"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"io/ioutil"
 	"log"
 	"net/http"
 	"os"
-	"strconv"
-
-	"github.com/joho/godotenv"
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-/*
-<map>
-  <entry>
-    <string>101af57f-f26c-40d3-86a3-309e74b93512</string>
-    <string>Send-Email-Notification</string>
-  </entry>
-</map>
-*/
-type ChannelIdNameMap struct {
-	XMLName xml.Name       `xml:"map"`
-	Entries []ChannelEntry `xml:"entry"`
+type ChannelStatusMap struct {
```

Comment:
Another general comment, this file has extensive use of channels, I do not understand much about prometheus or what the functionality here does, but I do know that typically channels are used to communicate between go routines. Are there go routines here that I do not see? (maybe in the prometheus stuff). If not why the use of channels?

Reply:
There are no go routines or channels. This is a prometheus exporter reporting on the status of channels in Mirth Connect. Channels are a central concept in Mirth. If this is confusing to a go developer, I could rename it to MirthChannelStatusMap or the like.